components/backup-stream/src/endpoint.rs

```diff
@@ -919,13 +921,30 @@ where
         }
     }
 
+    fn min_ts_worker(&self) -> future![()] {
+        let sched = self.scheduler.clone();
+        async move {
+            loop {
+                tokio::time::sleep(Duration::from_secs(10)).await;
```

Comment:
looks like it may cause too many check leader and increase cross az traffic for every 10 seconds.

Reply:
make it a configuration is better